src/pages/Components/Select/specs.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Specs'
+order: 3
+---
+
+## Anatomy
+
+=> anatomy image
+
+## Properties
+
+### CSS Tokens : layout
+
+| State       | Property      | Token                |
+| ----------- | ------------- | -------------------- |
+| **Default** | padding       | `mu-100` (1rem)      |
+| **Default** | border-radius | `radius.m` (4px)     |
+| **Default** | border-width  | `border.s` (1px)     |
+| **Default** | font-size     | `size.font.05`(16px) |
+
+### CSS Tokens : state colors
+
+> **You must always use the tokens, the colors are only there for that information.**
+
+| State                   | Property     | Token                              | Color   |
+| ----------------------- | ------------ | ---------------------------------- | ------- |
+| **Default**             | background   | `color.select.background`           | #ffffff |
+| **Default**             | border-color | `color.select.border`               | #6f676c |
+| **Default**             | color        | `color.select.text`                 | #222020 |
+| **Hover**               | border-color | `color.select.hover.border`         | #222020 |
```

Comment:
Why can't we use the `color.input.hover.border` already define ? It could be logical to always use the same variable for the same state for every type of form element.